build_tools/cmake/run_tf_tests.sh

```diff
@@ -19,14 +19,17 @@ IREE_LLVM_CPU_DISABLE="${IREE_LLVM_CPU_DISABLE:-0}"
 # Disable the tests by default to reduce the test time.
 IREE_VMVX_DISABLE="${IREE_VMVX_DISABLE:-1}"
 
+# TODO(scotttodd): use build_tools/pkgci/setup_venv.py here instead of BUILD_DIR
 source "${BUILD_DIR}/.env" && export PYTHONPATH
 source build_tools/cmake/setup_tf_python.sh
 
+python3 -m pip install lit
+LIT_SCRIPT=$(which lit)
```

Comment:
nit: Just for consistency with other places
```suggestion
LIT_SCRIPT="$(which lit)"
```